instructors.md

```diff
@@ -426,85 +427,70 @@ and have a rubric to follow.
-
-*   The discussion lead is free to use the Software Carpentry persistent
-    BlueJeans connection, or other options such as Google Hangouts or
-    Skype if they prefer.  The lead must post information about the
-    connection to the discussion session's Etherpad so that people
-    will know how to join.
-
-### During:
+The discussion session is meant to give students
+a chance to ask questions about the lesson materials in preparation for their final checkout,
+and to meet more people from the Carpentry community.
+The discussion lead's role is to give context on the lesson material,
+as each lesson goes through many revisions and discussions that cannot be easily seen in the finished product,
+and to evaluate whether trainees were adequately prepared for the discussion.
+
+Trainees do *not* do their 5-minute demonstration lessons during the discussion sessions.
+Instead,
+once they have completed the discussion to the leader's satisfaction,
+their names should be forwarded to the head of instructor training
+so that their demonstration lesson can be scheduled.
+
+As of Spring 2016,
+we are testing an alternative format in which trainees take part in workshop debriefing sessions
+rather than in bespoke lesson discussion sessions.
+The [checkout description](checkout.html) has details,
+and both formats are described below.
+
+### Before
+
+*   **Dedicated Lesson Discussion Sessions**
+    *   Send trainees a link to a signup Etherpad.
+    *   Set up a Google Hangout and mail the link to trainees.
+        Add the link to the signup Etherpad as well
+        (since it's the one place we know the trainees know to look).
+*   **Workshop Debriefing Sessions**
+    *   Send trainees a pointer to the
+        [Community Calendar](https://calendar.google.com/calendar/embed?src=oseuuoht0tvjbokgg3noh8c47g%40group.calendar.google.com).
+    *   Send them a link to the debriefing Etherpad.
+
+### During
 
 *   Remind students that the Etherpad is a public, persistent document,
     and that ideas and issues from the Etherpad may be used by the SCF.
-
-*   Explain how the checkout procedure itself goes:
-
-    *   After the trainee has completed the discussion session to the discussion leader's satisfaction,
-        the leader will forward her name to the trainer.
-
-    *   The trainer will then schedule a 5-minute slot for a demonstration lesson
-        and communicate it to the trainee.
-
-    *   In that session,
-        the trainee will be asked to teach 5 minutes from her chosen lesson
-        selected by the trainer.
-        The trainee will *not* know in advance which 5 minutes she will be asked to teach
-        (which encourages her to prepare the entire lesson).
-
-    *   Everyone will be notified afterwards by email whether they
-        passed or not.
-
+*   Briefly review the [checkout procedure](checkout.html).
+    Make sure trainees understand that *they* pick the lesson,
+    but *we* pick the segment from that lesson.
```

Comment:
Does the "we pick the segment" really apply to the discussion session? It's about the demo only?